cypress/e2e/CMS/$MiniLandingPageTest.feature

```diff
@@ -30,6 +30,10 @@ Feature: As a cms user I want to be able to create Mini Landing page content typ
         And user selects "Updated Date" checkbox
         And user selects "Include in search" from "Search Engine Restrictions" dropdown
         When user saves the content page
+        And page title is "Automated Test Mini Landing Page"
```

Comment:
took a sneak peak and I am glad I did ...

you cannot selectively chose which steps from your story to copy paste and which not
You copy your entire ticket's AC and you paste it underneath existing scenarios

Reply:
ok, thanks